pkg/catalog/manager/traverse.go

```diff
@@ -129,7 +147,28 @@ func traverseFiles(repoPath string, catalog *v3.Catalog) ([]v3.Template, []strin
 		template.Spec.CatalogID = catalog.Name
 		template.Name = fmt.Sprintf("%s-%s", catalog.Name, template.Spec.FolderName)
 
-		templates = append(templates, template)
+		v3.CatalogConditionRefreshed.Unknown(catalog)
```

Comment:
Is status "unknown" correct here? I think "False" is the right state

Reply:
I think False goes to "transitioning: error" state while Unknown goes to "transition: yes"